Whiskey/Functions/Invoke-WhiskeyNUnit2Task.ps1

```diff
@@ -47,6 +47,18 @@ function Invoke-WhiskeyNUnit2Task
         $reportGeneratorVersionArg['Version'] = $TaskParameter['ReportGeneratorVersion']
     }
 
+    $openCoverArgs = @()
+    if( $TaskParameter['OpenCoverArgument'] )
+    {
+        $openCoverArgs += $TaskParameter['OpenCoverArgument'] | ForEach-Object {if( $_.StartsWith('-')) { $_ } else { ('-' + $_) } }

```

Comment:
I had previously used @() instead of @{} for the custom parameters. I've updated this so and it's no splatting the values correctly..